packages/core/src/metrics/impl/AppMetrics.ts

```diff
@@ -0,0 +1,21 @@
+import {Metrics} from '../Metrics';
+
+/**
+ * Collect application metrics:
+ *
+ *  - appId - application bundle id
+ *  - appVersion - version of the application
+ *  - sdkVersion - AeroGear Services SDK version
+ */
+export class AppMetrics implements Metrics {
+  public identifier: string = 'app';
+  public collect(): any {
+    // TODO - does this even make sense to collect
+    const data: any = {
+      'appId': 1,
```

Comment:
This will make sense for cordova/react. There is not much sense to have this metrics for web.